src/components/views/settings/devices/LoginWithQRSection.tsx

```diff
@@ -15,15 +15,21 @@ limitations under the License.
 */
 
 import React from "react";
+import {
+    IMSC3882GetLoginTokenCapability,
+    IServerVersions,
+    UNSTABLE_MSC3882_CAPABILITY,
+} from "matrix-js-sdk/src/matrix";
 
-import type { IServerVersions } from "matrix-js-sdk/src/matrix";
 import { _t } from "../../../../languageHandler";
 import AccessibleButton from "../../elements/AccessibleButton";
 import SettingsSubsection from "../shared/SettingsSubsection";
 
 interface IProps {
     onShowQr: () => void;
     versions?: IServerVersions;
+    // we can't use the capabilities type from the js-sdk because it isn't exported
+    capabilities?: Record<string, any>;
```

Comment:
Is there anything to stop us from exporting it?

Reply:
> I can make another PR for this if you like?

:+1: Same name would just be an inconvenience by using an alias. But if creating a new export you can just rename it to something more specific.

Reply:
I've created https://github.com/matrix-org/matrix-js-sdk/pull/3266 to do the export.